sonar-csharp-plugin/src/main/java/org/sonar/plugins/csharp/CSharpSonarRulesDefinition.java

```diff
@@ -19,6 +19,7 @@
  */
 package org.sonar.plugins.csharp;
 
+import org.sonar.api.SonarRuntime;
```

Comment:
I am not super familiar with java styling convention but it seems that the other imports are ordered. This one doesn't seem to be.

Reply:
yes it is :trollface: